mod-source-record-manager-server/src/main/java/org/folio/dao/EventProcessedDaoImpl.java

```diff
@@ -19,11 +19,18 @@ public class EventProcessedDaoImpl implements EventProcessedDao {
 
   private static final Logger LOGGER = LogManager.getLogger();
 
-  private static final String TABLE_NAME = "events_processed";
+  public static final String EVENTS_PROCESSED_TABLE_NAME = "events_processed";
+  public static final String FLOW_CONTROL_EVENTS_COUNTER_TABLE_NAME = "flow_control_events_counter";
 
   private static final String INSERT_SQL = "INSERT INTO %s.%s (handler_id, event_id) VALUES ($1, $2)";
 
-  private PostgresClientFactory pgClientFactory;
+  private static final String INSERT_EVENT_AND_DECREASE_FLOW_CONTROL_COUNTER_SQL = "SELECT save_event_and_decrease_flow_control_counter($1, $2)";
+
+  private static final String DECREASE_COUNTER_SQL = "UPDATE %s.%s SET events_to_process = events_to_process - $1 RETURNING *";
```

Comment:
thinking you should reference the specific column instead of "*". If additional column is added to the table, it will not break the receiving code in the module. I get that this is a generalized query that could be for multiple tables but i think the schema name should be substituted but the table name should be constant.

Reply:
agree, changed code to return specific field